Sources/MountebankSwift/Models/Stub/Stub.swift

```diff
@@ -33,4 +36,20 @@ public struct Stub: Equatable {
     public init(response: any StubResponse, predicates: [Predicate]) {
         self.init(responses: [response], predicates: predicates)
     }
+
+    public init(responses: [any StubResponse], predicates requests: [Request]) {
+        self.init(responses: responses, predicates: requests.map { .equals($0) })
+    }
+
+    public init(response: any StubResponse, predicate request: Request) {
+        self.init(response: response, predicate: .equals(request))
+    }
+
+    public init(responses: [any StubResponse], predicate request: Request) {
+        self.init(responses: responses, predicate: .equals(request))
+    }
+
+    public init(response: any StubResponse, predicates requests: [Request]) {
+        self.init(response: response, predicates: requests.map { .equals($0) })
+    }
```

Comment:
I noticed that for the ui tests for our app we only use the `.equals` predicate yet and I think this will always be the most used predicate. So I've added 4 convenience methods for when just using a simple request. These all map to `.equals`

Usage: 

```
Stub(
    responses: Is(body: "Hello world"),
    predicate: Request(
        method: .get,
        path: "/some/test/path"
    )
)
```

Instead of: 

```
Stub(
    responses: Is(body: "Hello world"),
    predicate: .equals(
        Request(
            method: .get,
            path: "/some/test/path"
        )
    )
)
```

Do we want this? 

Does it make sense still call the parameter: `predicate:` ? 
- I considered naming it `request:` but, it is in the end still the predicate.. 
- I also considered calling it `predicateEquals` to differentiate, but these methods are created to make the usage as easy as possible, so the shorter the better.